libcontainer/integration/checkpoint_test.go

```diff
@@ -61,6 +62,12 @@ func testCheckpoint(t *testing.T, userns bool) {
 		t.Skipf("criu binary not found: %v", err)
 	}
 
+	// Workaround for https://github.com/opencontainers/runc/issues/3532.
+	out, err := exec.Command("rpm", "-q", "criu").CombinedOutput()
```

Comment:
This might not work in Debian-based distributions such as Ubuntu.

Reply:
I think that is on purpose. 

If rpm doesn't exist then the output won't match the regex and that is okay, ubuntu doesn't have this criu issue. We just need to skip it in very specific cases, this will work on those cases and not fail the test in the cases we don't need to skip it.

Reply:
That is probably why all the tests passed here too :)

Reply:
This is why the code doesn't do anything if there is an error from this command. Since we only need this workaround for  CentOS Stream 9 which we can safely assume has working rpm.